model/simulationtests/daylighting_devices.rb

```diff
@@ -31,16 +31,16 @@
 # add design days to the model (Chicago)
 model.add_design_days
 
-thermal_zones = model.getThermalZones.sort_by { |tz| tz.name.to_s }
+thermal_zones = model.getThermalZones.sort_by(&:nameString)
 thermal_zone = thermal_zones[0]
 thermal_zone.setUseIdealAirLoads(true)
-spaces = thermal_zone.spaces.sort_by { |s| s.name.to_s }
+spaces = thermal_zone.spaces.sort_by(&:nameString)
 
 windows = []
 spaces.each do |space|
-  surfaces = space.surfaces.sort_by { |s| s.name.to_s }
+  surfaces = space.surfaces.sort_by(&:azimuth)
```

Comment:
The surfaces are not in a consistent naming convention, so instead we sort by azimuth, which ensures we always take for eg the East wall, and not randomly the West, North, South one